Mathlib/Order/LiminfLimsup.lean

```diff
@@ -111,6 +112,22 @@ theorem IsBoundedUnder.comp {l : Filter γ} {q : β → β → Prop} {u : γ →
 section Preorder
 variable [Preorder α] {f : Filter β} {u : β → α} {s : Set β}
 
+lemma IsBoundedUnder.eventually_le (h : IsBoundedUnder (· ≤ ·) f u) :
+    ∃ a, ∀ᶠ x in f, u x ≤ a := by
+  obtain ⟨a, ha⟩ := h
+  use a
+  exact eventually_map.1 ha
+
+lemma IsBoundedUnder.eventually_ge (h : IsBoundedUnder (· ≥ ·) f u) :
+    ∃ a, ∀ᶠ x in f, a ≤ u x :=
+  IsBoundedUnder.eventually_le (α := αᵒᵈ) h
+
+lemma isBoundedUnder_of_eventually_le {a : α} (h : ∀ᶠ x in f, u x ≤ a) :
+    IsBoundedUnder (· ≤ ·) f u := ⟨a, h⟩
+
+lemma isBoundedUnder_of_eventually_ge {a : α} (h : ∀ᶠ x in f, a ≤ u x) :
+    IsBoundedUnder (· ≥ ·) f u := ⟨a, h⟩
```

Comment:
```suggestion
lemma IsBoundedUnder.eventually_le (h : IsBoundedUnder (· ≤ ·) f u) : ∃ a, ∀ᶠ x in f, u x ≤ a := h
lemma IsBoundedUnder.eventually_ge (h : IsBoundedUnder (· ≥ ·) f u) : ∃ a, ∀ᶠ x in f, a ≤ u x := h

lemma isBoundedUnder_of_eventually_le {a : α} (h : ∀ᶠ x in f, u x ≤ a) :
    IsBoundedUnder (· ≤ ·) f u := ⟨a, h⟩

lemma isBoundedUnder_of_eventually_ge {a : α} (h : ∀ᶠ x in f, a ≤ u x) :
    IsBoundedUnder (· ≥ ·) f u := ⟨a, h⟩
```
Do you really think we need those four lemmas? They are all true by definition

Reply:
> They are all true by definition

Not exactly. This works with `IsBounded`, but `IsBoundedUnder` has an additional layer. For instance, something like

`(hu : f.IsBoundedUnder (· ≤ ·) u); obtain ⟨U, hU⟩ := hu`

yields `hU : ∀ᶠ (x : R) in map u f, (fun x1 x2 ↦ x1 ≤ x2) x U`, while

`(hu : f.IsBoundedUnder (· ≤ ·) u); obtain ⟨U, hU⟩ := hu.eventually_le`

yields `hU : ∀ᶠ (x : α) in f, u x ≤ U`, which is more readily useful. There are a couple instances in this new version of `Mathlib.Order.LiminfLimsup` where this is useful (see e.g. the proof of `isCoboundedUnder_ge_add`).

This is not a big change, but I did not find a simple clean solution, and a basic lemma with a dot notation felt intuitive enough.